packages/react/src/components/Modal/index.tsx

```diff
@@ -224,7 +228,7 @@ const ModalBackground = animated(styled.div<{
   }
 `)
 
-const ModalCrossButton = styled(IconButton)`
+export const ModalCloseButton = styled(IconButton)`
```

Comment:
In case user want to set `isDismissable` to false and handle it themselves, for example, pass different `onClose` for close button and background.